services/web/app.py

```diff
@@ -175,10 +175,12 @@ def auth_saml(self, auth):
             app.logger.debug(f"SAML auth_data: {auth_data}")
             predicate = auth["saml"]
             authorized = eval(predicate, dict(user=auth_data))
+            multi_value_attrs = os.environ.get("MULTI_VALUE_SAML_ATTRS", "").split(",")
```

Comment:
Er, this probably belongs in `config.py`, not the environment :)

Reply:
Blast! That's where I had it initially, and then I got in my own head. 😆 I'll move it back, thanks!